### PR TITLE
Create chatflows workflow

### PR DESCRIPTION
**Pull request by AhmedHanafy725**
Add workflow to run chatflows tests with each pre-release.

### CODE REVIEW

**Comment by zaibon** [comment]
2 things here.
1. Can you describe exactly what this workflow does and what is the impact on the network when it runs ?

2. I would like to change the trigger event of this workflow from creation of pre-release to a manual one. See: https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/manually-running-a-workflow

@muhamadazmy you thought on this ?

File: .github/workflows/chatflows.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          sudo poetry install
+      - name: Run tests
+        env:
+          TNAME: ${{ secrets.TNAME }}
+          EMAIL: ${{ secrets.EMAIL }}
+          WORDS: ${{ secrets.WORDS }}
+          WALLET_SECRET: ${{ secrets.WALLET_SECRET }}
+        run: |
+          cd js-sdk
+          sudo --preserve-env=TNAME --preserve-env=EMAIL --preserve-env=WORDS --preserve-env=WALLET_SECRET \
```
Comment (muhamadazmy) [inline .github/workflows/chatflows.yml:32]: i think it should be $WORDS, and $WALLET_SECRET also why not do that directly via the ${{ secrets.WORDS }} directly in the run command ?

**Comment by muhamadazmy** [comment]
While this indeed useful but an automated workflow is not the right way to do it. This will run DURING the pre-release building which mans the testnet still did not get this pre-release. hence the tests are running against an older release that is currently on testnet.

The right way imho to do it is by running it manually hence it does not even have to be part of zos repo

File: .github/workflows/chatflows.yml (hunk 1 above)
Comment (AhmedHanafy725) [reply .github/workflows/chatflows.yml:32]: I can't use Github secrets while using `sudo`, so I take them to `sudo` using their names without `$`

**Comment by AhmedHanafy725** [comment]
The workflow is changed to run manually.
This workflow is running most of the dashboard solutions, marketplace, and threebot deployer chatflows.

**Pull request merged**